refactor/queuer/openapi.yaml

```diff
@@ -0,0 +1,80 @@
+openapi: 3.0.2
+info:
+  title: Covalent Queuer Service API
+  version: 0.1.0
+paths:
+  /api/v0/submit/dispatch:
+    post:
+      tags:
+        - Submit
+      summary: Submit Workflow
+      description: Submit a workflow
+      operationId: submit_workflow_api_v0_submit_dispatch_post
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ResultPickle'
```

Comment:
Is this the right schema to submit to the queuer? We're submitting a workflow, not a result... I think?

Reply:
ok, can that be included in the description of the API method? otherwise I think this is sort of confusing
<img width="1440" alt="Screen Shot 2022-03-07 at 4 33 49 PM" src="https://user-images.githubusercontent.com/25555884/157121504-81441873-d514-4038-82e4-c5ca9f254652.png">

Reply:
Implemented.